subprojects/docs/src/docs/userguide/developing-plugins/publishing_gradle_plugins.adoc

```diff
@@ -115,6 +122,7 @@ Here is an example of the `pluginBundle` configuration that you can use for the
 pluginBundle {
     website = 'https://www.gradle.org/'
     vcsUrl = 'https://github.com/gradle/greeting-plugin-example'
+    description = 'Template for people to start their own plugin adventure'
```

Comment:
I do not understand this change. I thought the plan was to have all plugin metadata in the `gradlePlugins` block

Reply:
I've made the change to illustrate that the description can also be specified globally, for the entire bundle, not just individually for each published plugin. Do we want to deprecate that capability?